release-notes/release-notes-insiders.json

```diff
@@ -1,4 +1,36 @@
 [
+    {
+        "version": "4.7.0-insiders",
+        "title": "Fiddler Everywhere v4.7.0",
+        "date": "September 27, 2023",
+        "items": [
+            {
+                "type": "New",
+                "entries": [
+                    {
+                        "title": "Add option to set custom chromium flags or to start an entirely custom chromium browser instance"
```

Comment:
I think "Add option to persist preconfigured browser data" represents more accurately what is the new behaviour introduced with the story.